src/battle-animations.ts

```diff
@@ -625,7 +625,7 @@ class BattleScene {
 		pokemonhtml = '<div class="teamicons">' + pokemonhtml + '</div>';
 		const $sidebar = (side.n ? this.$rightbar : this.$leftbar);
 		if (side.name) {
-			$sidebar.html('<div class="trainer"><strong>' + BattleLog.escapeHTML(side.name) + '</strong><div class="trainersprite" style="background-image:url(' + Dex.resolveAvatar(side.avatar) + ')"></div>' + pokemonhtml + '</div>');
+			$sidebar.html('<div class="trainer" ' + (side.rating ? 'title="Rating: ' + side.rating + "\">" : '>') + '<strong>' + BattleLog.escapeHTML(side.name) + '</strong><div class="trainersprite" style="background-image:url(' + Dex.resolveAvatar(side.avatar) + ')"></div>' + pokemonhtml + '</div>');
```

Comment:
Native `title` attributes generally don't work on mobile because you can't hover. Does the tooltip logic PS uses on mobile work based on `title`? If not, perhaps we want to be the PS tooltips instead

Reply:
PS tooltips are extremely hard to use, sadly.

Reply:
So our story on mobile here is "tough luck"? I don't know how many power users who care about ELO of each of their opponents are also using PS on a touch device, so maybe its an irrelevant concern?

Reply:
I guess the only other solutions is adding hacky jQuery to show a message on-click? But then the box probably wouldn't be at the place of the avatar so we would need to also fix that. I don't think it's _too_ worth it.

Reply:
I think its probably fine to punt.

Reply:
It's not _completely_ unusable on mobile, I think it appears on the long-press menu?

Reply:
For the record, I know PS tooltips are hard to use because I've wanted to use them for everything else ever (appearing instantly instead of on a delay is really good for UX).

Maybe I could write some code replacing all titles with PS tooltips...

Reply:
Some tooltip libraries do that, which is why I asked. As for long press - was not aware of that, but I'll take your word for it.